### PR TITLE
chore: check if private keys and addresses are matched

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

File: example/src/main/java/org/nervos/ckb/SingleKeySingleSigTxExample.java [hunk 1]
```diff
@@ -33,6 +36,13 @@
   }
 
   public static void main(String[] args) throws Exception {
+    AddressUtils utils = new AddressUtils(Network.TESTNET);
+    String testPublicKey = ECKeyPair.publicKeyFromPrivate(TestPrivateKey);
+    if (!TestAddress.equals(utils.generateFromPublicKey(testPublicKey))) {
+      System.out.println(
+          "private key " + TestPrivateKey + " and address " + TestAddress + " are not match");
```
Comment (duanyytop) [inline example/src/main/java/org/nervos/ckb/SingleKeySingleSigTxExample.java:43]: Must not print private key

Comment (shaojunda) [reply example/src/main/java/org/nervos/ckb/SingleKeySingleSigTxExample.java:43]: https://github.com/nervosnetwork/ckb-sdk-java/pull/351/files/ba349e17a157811af5764b11886f420f7ba76300#diff-71ff690890b485a4a8e017d78306c475L23 just print example private key, Will there be any harm?

Comment (shaojunda) [reply example/src/main/java/org/nervos/ckb/SingleKeySingleSigTxExample.java:43]: got it, I will fix this.

Comment (shaojunda) [reply example/src/main/java/org/nervos/ckb/SingleKeySingleSigTxExample.java]: updated: [`b4b30c5` (#351)](https://github.com/nervosnetwork/ckb-sdk-java/pull/351/commits/b4b30c5f120a43e8ba561108710a1c5259c4aef5)

Comment (duanyytop) [reply example/src/main/java/org/nervos/ckb/SingleKeySingleSigTxExample.java]: I think `System.out.println("Private key and address (" + TestAddress + ") are not match");` is OK.

Comment (shaojunda) [reply example/src/main/java/org/nervos/ckb/SingleKeySingleSigTxExample.java]: The last four characters of the private key are added for the convenience of debugging when there are multiply addresses

Comment (duanyytop) [reply example/src/main/java/org/nervos/ckb/SingleKeySingleSigTxExample.java]: I think security is more important than debugging convenience and the probability of this error is very small.

**Review by duanyytop — APPROVED** [review]

File: example/src/main/java/org/nervos/ckb/SingleKeySingleSigTxExample.java [hunk 2]
```diff
@@ -33,6 +36,12 @@
   }
 
   public static void main(String[] args) throws Exception {
+    AddressUtils utils = new AddressUtils(Network.TESTNET);
+    String testPublicKey = ECKeyPair.publicKeyFromPrivate(TestPrivateKey);
+    if (!TestAddress.equals(utils.generateFromPublicKey(testPublicKey))) {
+      System.out.println("private key " + "and address " + TestAddress + " are not matched");
```
Comment (duanyytop) [inline example/src/main/java/org/nervos/ckb/SingleKeySingleSigTxExample.java:42]:
```suggestion
      System.out.println("Private key and address " + TestAddress + " are not matched");
```

File: example/src/main/java/org/nervos/ckb/SingleSigWithCkbIndexerTxExample.java [hunk 3]
```diff
@@ -45,6 +48,16 @@
   }
 
   public static void main(String[] args) throws Exception {
+    AddressUtils utils = new AddressUtils(Network.TESTNET);
+    for (int i = 0; i < SendAddresses.size() - 1; i++) {
+      String privateKey = SendPrivateKeys.get(i);
+      String testPublicKey = ECKeyPair.publicKeyFromPrivate(SendPrivateKeys.get(i));
+      String address = SendAddresses.get(i);
+      if (!address.equals(utils.generateFromPublicKey(testPublicKey))) {
+        System.out.println("private key " + "and address " + address + " are not matched");
```
Comment (duanyytop) [inline example/src/main/java/org/nervos/ckb/SingleSigWithCkbIndexerTxExample.java:57]:
```suggestion
        System.out.println("Private key and address " + address + " are not matched");
```

File: example/src/main/java/org/nervos/ckb/SingleSigWithIndexerTxExample.java [hunk 4]
```diff
@@ -45,6 +48,16 @@
    * collect live cells, you should await some time after calling index_lock_hash rpc.
    */
   public static void main(String[] args) throws Exception {
+    AddressUtils utils = new AddressUtils(Network.TESTNET);
+    for (int i = 0; i < SendAddresses.size() - 1; i++) {
+      String privateKey = SendPrivateKeys.get(i);
+      String testPublicKey = ECKeyPair.publicKeyFromPrivate(SendPrivateKeys.get(i));
+      String address = SendAddresses.get(i);
+      if (!address.equals(utils.generateFromPublicKey(testPublicKey))) {
+        System.out.println("private key " + "and address " + address + " are not matched");
```
Comment (duanyytop) [inline example/src/main/java/org/nervos/ckb/SingleSigWithIndexerTxExample.java:57]:
```suggestion
        System.out.println("Private key and address " + address + " are not matched");
```

**Review by duanyytop — APPROVED** [review]

**Pull request merged**